introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone_dist_script_mode.ipynb

```diff
@@ -48,7 +48,7 @@
    "outputs": [],
    "source": [
     "import sys\n",
-    "!{sys.executable} -m pip install -qU awscli boto3 \"sagemaker>=1.71.0,<2.0.0\""
+    "!{sys.executable} -m pip install -qU awscli boto3 \"sagemaker==2.*\""
```

Comment:
Please refer the [template for upgrade/downgrade approach](https://github.com/aws/amazon-sagemaker-examples/blob/master/template.ipynb) that won't mess up the kernel.